File: Ana/Ana/models/Imc.cs
namespace Ana.Models;

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

public class Imc
{
    public int Id { get; set; }
    public int AlunoId { get; set; }
    public double Altura { get; set; }
    public double Peso { get; set; }
    public double IMC { get; set; }
    public string Classificacao { get; set; }
    public string DataNascimento  { get; set;}
    
    
    [ForeignKey("AlunoId")]
    public Aluno Aluno { get; set; }
}

File: front/src/models/imc.ts
import { Aluno } from "./aluno";

export interface IMC {
  id: number;
  alunoId: number;
  altura: number;
  peso: number;
  imc: number;
  classificacao: string;
  dataCriacao: string;
  aluno: Aluno;
}
